src/org/skypro/skyshop/Search/SearchEngine.java

```diff
@@ -2,25 +2,24 @@
 
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Map;
+import java.util.TreeMap;
 
 public class SearchEngine {
-    List<Searchable> searchables;
+    private final List<Searchable> searchables;
 
     public SearchEngine() {
         searchables = new ArrayList<>();
     }
 
-    public void searchInfo(String term) {
-        List<Searchable> info = new ArrayList<>();
+    public Map<String, Searchable> searchInfo(String term) {
+        Map<String, Searchable> info = new TreeMap();
```

Comment:
Не забываем про дженерики
```suggestion
        Map<String, Searchable> info = new TreeMap<>();
```